src/components/RedigerAvansertBeregning/RedigerAvansertBeregning.tsx

```diff
@@ -526,9 +526,22 @@ export const RedigerAvansertBeregning: React.FC<{
                 }
               })
               setTemporaryHelUttak((prevState) => {
+                const sluttAlderMonths =
```

Comment:
Noen spesiell grunn til å bruke engelsk "Months" her?

Reply:
Jeg prøvde å skille fra "maaneder" som vi bruker ellers for måneden som telles fra fødselsdato. Dette her var alderen "konvertert" i antall måneder.
Jeg kaller heller det for sluttAlderAntallMaaneder. Det er uansett en lokal scoped variabel så jeg bør ikke legge for mye i det.